src/Uno.UI/UI/Xaml/Shapes/Shape.Android.cs

```diff
@@ -34,19 +34,6 @@ protected override void OnDraw(Canvas canvas)
 				return;
 			}
 
-			//Drawing paths on the canvas does not respect the canvas' ClipBounds
```

Comment:
It would be best to include additional regression tests.

Reply:
@jeromelaban I wasn't able to isolate a minimal repro for testing.

Reply:
Well you could test the `ClosedFigure` sample you found in the screenshot diff https://github.com/unoplatform/uno/pull/13490#issuecomment-1704608537 and validate that you have black pixels in the "title bar" of the page.

Reply:
@dr1rrb Somehow, I'm not seeing the new behavior locally. So `ClosedFigure` is still broken :/

Reply:
@Youssef1313 is there something left to do here?

Reply:
Ideally, a test, but since I couldn't figure out one, I think it should be okay to merge as is.